content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md

```diff
@@ -38,7 +38,7 @@ the container starts.
 
 1. Create a Pod based on the YAML configuration file:
 
-       kubectl create -f https://k8s.io/docs/tasks/debug-application-cluster/termination.yaml
+        kubectl create -f https://raw.githubusercontent.com/kubernetes/website/master/content/en/docs/tasks/debug-application-cluster/termination.yaml
```

Comment:
Why this change?

Reply:
The same as  PR[https://github.com/kubernetes/website/pull/6435](https://github.com/kubernetes/website/pull/6435)

Reply:
I am curious why the old url doesn't work.

Reply:
the old url works in my env... curious.

Reply:
Please don't change URL of the yaml file. If this isn't working, then we need to get the yaml file in the correct directory.

Reply:
fixed, PTAL